### ISSUE TITLE
Is it an actual tutorial?

### ISSUE THREAD

**Issue by vi** [tailhook/tokio-tutorial#1]
I see some examples, but no introduction and other text, pictures and so on...

**Comment by tailhook** [comment]
No, this was a bunch of examples for the session called "tokio tutorial" not something that contains tutorial text. Mostly they are already out of date I think.
